oap-server/server-fetcher-plugin/fetcher-proto/src/main/proto/cilium/hubble/relay/relay.proto

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
```

Comment:
I think we should update the LICENSE of source codes, as we copied these from a new project.

Reply:
I have update the LICENSE file, please check.